doc/docs.md

```diff
@@ -157,7 +157,6 @@ To do so, run the command `v up`.
     * [Hot code reloading](#hot-code-reloading)
     * [Cross compilation](#cross-compilation)
     * [Cross-platform shell scripts in V](#cross-platform-shell-scripts-in-v)
-	* [Vsh scripts with no extension](#vsh-scripts-with-no-extension)
```

Comment:
I do not get it - why remove the title?

Also they are not cross platform - they depend on the so called shebang syntax, that is only supported on *nix systems, but not on windows, so that section logically should not be a subsection of the `[Cross-platform shell scripts in V]`.